tendrl/commons/__init__.py

```diff
@@ -96,7 +96,7 @@ def _validate_ns_obj_definitions(self, raw_ns, defs):
                 if self._get_atoms(obj_name):
                     LOG.info("Validating registered (.py) atoms in %s.objects.%s.atoms", raw_ns, obj_name)
                     defined_atoms = defined_objs.get(obj_name, {}).get("atoms", {})
-                    regd_atoms = [atom_name for atom_name in self._get_atoms(obj_name) if not self.get_atom(obj_name,atom_name).internal]
+                    regd_atoms = [atom_name for atom_name in self._get_atoms(obj_name) if not hasattr(self.get_atom(obj_name,atom_name), "internal") and atom_name != "BaseAtom"]
```

Comment:
Marking BaseAtom, BaseFlow and BaseObject asks for `_defs` attribute as well. What should be set for the same?
Setting `_defs = {}` is for sure not correct I feel.